src/lib.rs

```diff
@@ -27,50 +27,6 @@
 //!   double-key variants of Schnorr signatures.
 //! - `gadgets`: Provides zero-knowledge gadgets for use in circuits. Available
 //!   only when compiled with the `alloc` feature.
-//!
-//! ## Examples
```

Comment:
I still think it is nice to have a doc test here that showcases the primary use of the crate.

Reply:
The doc tests are with the `Signature` and `DoubleSignature` structs. I only removed the duplicates